GoogleUtilities/ISASwizzler/GULSwizzledObject.m

```diff
@@ -61,4 +65,23 @@ - (BOOL)respondsToSelector:(SEL)aSelector {
   return [gulClass instancesRespondToSelector:aSelector] || [super respondsToSelector:aSelector];
 }
 
+- (void)dealloc {
+  @autoreleasepool {
+    // Set the generated class to the original class.
+    Class originalClass = [[self gul_class] superclass];
+    if (![originalClass superclass] || [self isProxy]) {
+      return;
+    }
+
+    // `self` is now the original class.
+    object_setClass(self, originalClass);
```

Comment:
I'm curious if zombie diagnostic still works after this.

Reply:
I pulled this branch into the Performance quickstart and the quickstart did not crash (both with and without zombies enabled). Is this what you are referring to?